roles/common/tasks/main.yml

```diff
@@ -104,3 +104,9 @@
 
 - include: monitoring.yml tags=monitoring,common
   when: monitoring.enabled|default('True')|bool
+
+- include: audit-logging.yml
+  tags:
+    - logrotate
+    - logging
+  when: logging.enabled|default('True')|bool
```

Comment:
This will actually capture too much. When logging is disabled, logrotate will not be installed which we likely always want.